frontend/components/snippet.tsx

```diff
@@ -15,22 +18,39 @@ const CodeContainer = styled.span`
       max-width: 40%;
       overflow: hidden;
       text-overflow: ellipsis;
-    `};
```

Comment:
👀 surprised our linter didn't catch this trailing semi before

Reply:
Yea, not sure! Maybe because it's inside a (`styled`) tagged template literal?